kin-sdk-core/src/androidTest/java/kin/sdk/core/RequestTest.java

```diff
@@ -0,0 +1,134 @@
+package kin.sdk.core;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import android.os.Looper;
+import android.support.test.runner.AndroidJUnit4;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLongArray;
+import java.util.function.Consumer;
+import junit.framework.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class RequestTest {
+
+    private static final int TASK_DURATION_MILLIS = 50;
```

Comment:
can it be less than 50?

Reply:
I tried now with 10 (and 20 for the timeout) and it worked, but the risk is flaky tests (my machine and tested emulator is probably faster than travis ones), I can't say where's the limit but better not to take the risk, tests now took ~200ms, so it's negligible.

Reply:
for now, it's ok but if we'll create more `Request` tests, then it would take time much more time for a complete test cycle to be done.
👍